docs/bdd_integration.rst

```diff
@@ -192,3 +214,117 @@ functions or check the :ref:`dataset <dataset>` module for more implementation d
 * :code:`[ENV:xxxx]`: Value of the OS environment variable xxxx
 * :code:`[FILE:xxxx]`: String with the content of the file in the path xxxx
 * :code:`[BASE64:xxxx]`: String with the base64 representation of the file content in the path xxxx
+
+This feature can be customized by defining your own map_param function, the same way as for replace_param, but again,
+it is something not recommended unless very well justified.
+
+In order to apply string replacements at code level, the following functions are available in the context:
+
+.. code:: console
+
+    context.map_param(parameter)
+    context.replace_param(parameter)
+
+These replacements and mappings are automatically applied to all the steps of the Toolium's catalogue.
+
+The configuration and context replacements are made first, followed by the rest of variable replacements (i.e.
+`map_param` is always executed before `replace_param`), and they apply to all the possible parameters of the step:
+
+* context.text
+* context.table (each of the cells within the table)
+* parameters of the function that implements the step, e.g. `param1` and `param2` in `my_step_impl(context, param1, param2)`
+
+If you implement your own steps, remember to use the `@step` decorator implemented by Toolium to take advantage of
+this feature:
+
+.. code:: console
+
+    from toolium_telefonica.behave import step
```

Comment:
remove references to toolium_telefonica, in this case, behave import can be used